codingQuestion.py

```diff
@@ -1 +1,12 @@
-
+L = int(input())
+N = int(input())
+
+for _ in range(N):
+    W, H = map(int, input().split())
+    
+    if W < L or H < L:
+        print("UPLOAD ANOTHER")
+    elif W == H:
+        print("ACCEPTED")
+    else:

```

Comment:
Indentation of this line is incorrect. It should be indented to be part of the 'else' block.